packages/fiori/src/NotificationAction.ts

```diff
@@ -0,0 +1,97 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import ButtonDesign from "@ui5/webcomponents/dist/types/ButtonDesign.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+
+type NotificationActionClickEventDetail = {
+	targetRef: NotificationAction | null,
```

Comment:
the type here should be a ui5-button. targetRef is the button that is rendered by all the various notification*.hbs templates.

![image](https://user-images.githubusercontent.com/22086365/220933197-7b502074-f608-4fc7-8f12-6e1f78474722.png)